modular_skyrat/modules/modular_items/lewd_items/code/lewd_arousal/arousal.dm

```diff
@@ -50,6 +50,15 @@
 	return TRUE
 
 
+
+/mob/living/carbon/human/Life(seconds_per_tick, times_fired)
+	. = ..()
+	if (keep_arousal_on_goal && (arousal + 5 < arousal_goal))
+		adjust_arousal(5)
+	else if (keep_arousal_on_goal && (arousal < arousal_goal))
+		adjust_arousal(1)
```

Comment:
tbh it'd be better to just have a proc that registers to Life instead and calls this code, instead of extending a critical proc like Life